### DISCUSSION TITLE
<title>Contributions in the table on my profile are not being updated..</title>

### DISCUSSION THREAD

**Discussion by devMarcos2021** [community/community#66875] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I'm concerned that my contribution table on GitHub is not updating as it used to, even after sending and updating projects in my public repositories. This is reducing my visibility on the platform and, consequently, my job opportunities.

**Comment by JustOptimize** [comment]
To fix this, head to your profile page, in the top right of the contribution table you can enable Private Contributions.
Your updates to your public repositories are working as expected

**Reply by JustOptimize** [reply]
@devMarcos2021, if this solved your issue please mark this as the answer

**Reply by Jonak-Adipta-Kalita** [reply]
i feel bad for you...

**Reply by JustOptimize** [reply]
An he replies with emojis instead of actually marking the answer as an actual answer 🤣

**Reply by Jonak-Adipta-Kalita** [reply]
😭😂

**Comment by babayl0729** [comment]
I just noticed today that I'm having the same issue. @JustOptimize I tried your advice but no avail.

**Reply by JustOptimize** [reply]
Hi, this is how your history shows to others
![image](https://github.com/community/community/assets/54294419/ae85e196-785b-44d3-b8f0-7b6cb4192139)
do you have commits in private repositories that should appear?

**Reply by babayl0729** [reply]
Thanks for looking into this. I actually always commit to a public repo. Also I just checked my contribution table and it seems it is updated at least from yesterday. Probably the issue (or not really an issue), it didn't update right away. Usually right after I commit, the contribution table updates right away but it seems like this time it took some time to update. Thanks again!
